### PR TITLE
7 create final call to action cta section from figma design

### LINKED ISSUE #7: Create Final Call-to-Action (CTA) Section from Figma Design
**Description**
The aim of this issue is to create the Final Call-to-Action (CTA) Section for the landing page. The Final CTA Section serves as the concluding element, encouraging users to take a specific action i.e either to start a chat or join a chat.

**Visual Reference**
Final CTA

### PR DESCRIPTION
**Pull request by Oyinbrakemigrace**

### CODE REVIEW

**Comment by Oyinbrakemigrace** [comment]
finalCTA

**Review by Remi-dee — APPROVED** [review]
LG

File: app/components/FinalCTA_comp.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Image from "next/image";
+import chatIcon from "../../public/assets/icons/chat.png";
+import joinChatIcon from "../../public/assets/icons/joinChat.png";
+import finalCtaImg from "../../public/assets/images/final_cta_img_1.png";
+
+const FinalCTA_comp = () => {
```
Comment (annonymousauthority) [inline app/components/FinalCTA_comp.js]: Snake case isn't used for component naming

File: app/components/FinalCTA_comp.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              <p className="text-[#2C1FA3] font-medium">Start a Chat</p>
+            </button>
+            <button className="flex flex-row justify-center text-center bg-transparent border-[2px] border-white text-white px-9 py-2 space-x-2 rounded-90xl">
+              <Image src={joinChatIcon} />
+              <p className="font-medium ">Join a Chat</p>
+            </button>
+          </div>
+        </div>
+        <div>
+          <Image src={finalCtaImg} sizes={20} />
```
Comment (annonymousauthority) [inline app/components/FinalCTA_comp.js]: No `alt` attribute here

File: app/components/FinalCTA_comp.js (hunk 1 above)
Comment (annonymousauthority) [inline app/components/FinalCTA_comp.js]: Let's use a proper function for naming components not arrow function

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Changes required

**Review by Remi-dee — APPROVED** [review]

**Review by annonymousauthority — APPROVED** [review]
Seen

**Pull request merged**